### DISCUSSION TITLE
save the model and use variations

### DISCUSSION THREAD

**Discussion by xeoster** [lllyasviel/Fooocus#3764] [Q&A]
When we create a model, I want to save this model and make different variations later. How do I make such an option
![2024-11-29_18-10-11_9337](https://github.com/user-attachments/assets/145e07af-7a14-4233-8fc2-318372a5b252) like this model

**Comment by Yogesh-DevHub** [comment]
Use image to image generation / face swap to generate the same model again. If you do not feel the face is similar, then try generating multiple images using same model. Pick 10 - 15 best results and train a LoRA, then you will have a similar looking model all the time.
